src/cmd/ksh93/tests/getconf.sh

```diff
@@ -0,0 +1,101 @@
+# Tests for `getconf` builtin
+
+# ==========
+#   -a, --all       Call the native getconf(1) with option -a.
+actual=$(getconf -a)
```

Comment:
@krader1961 This test fails on macOS, so I presume `-a` option is not supported on macOS. I am going to disable it and merge this pull request.